t/134-worker-FFI.t

```diff
@@ -0,0 +1,116 @@
+# vim:set ft= ts=4 sw=4 et fdm=marker:
+use lib 'lib';
+use Test::Nginx::Socket::Lua;
+
+#worker_connections(1014);
+#master_on();
+workers(3);
+#log_level('warn');
+
+repeat_each(2);
+#repeat_each(1);
+
+plan tests => repeat_each() * (blocks() * 3);
+
+#no_diff();
+#no_long_string();
+run_tests();
+
+__DATA__
+
+=== TEST 1: content_by_lua + ngx_http_lua_ffi_worker_pid()
```

Comment:
BTW, to run ngx_lua's test suite with lua-resty-core enabled, we just need to set the following environment:

```
export TEST_NGINX_INIT_BY_LUA="package.path = '$PWD/../lua-resty-core/lib/?.lua;' .. (package.path or '') require 'resty.core'"
```

Thanks to the `Test::Nginx::Socket::Lua` test class that implements the magical `TEST_NGINX_INIT_BY_LUA` environment variable :)